### PR TITLE
[LAP-141] decal event

### PR DESCRIPTION
**Pull request by Algo-ryth-mix**

### CODE REVIEW

File: applications/sandbox/systems/event_based_explosion_system.hpp [hunk 1]
```diff
@@ -145,6 +162,7 @@ namespace ext
 
             case BOOM_STRENGTH::AN602:
             {
+                log::debug("WORLD ENDING BOOM");
```
Comment (Jelled1st) [inline applications/sandbox/systems/event_based_explosion_system.hpp:165]: lol

File: applications/sandbox/systems/testsystem.hpp [hunk 2]
```diff
@@ -725,6 +726,8 @@ class TestSystem final : public System<TestSystem>
             ent.add_components<rendering::mesh_renderable>(mesh_filter(uvsphereH.get_mesh()), rendering::mesh_renderer(copperH));
             ent.add_component<sah>({});
             ent.add_components<transform>(position(0, 3, -3.6f), rotation(), scale());
+
+            //ext::BigBoom::createExplosiveEntity({1.0f,1.0f,1.0f},aluminumH,ent);
```
Comment (Jelled1st) [inline applications/sandbox/systems/testsystem.hpp:730]: if this code is no longer required, it can perhaps be removed

**Review by Jelled1st — APPROVED** [review]

**Comment by GlynLeine** [comment]
@Algo-ryth-mix what's the state of this PR? is it still needed as an engine instead of as a school project?

**Comment by Algo-ryth-mix** [comment]
The feature is complete, but it can be dropped since it is no longer necessary

**Pull request closed without merging**